x/ccv/provider/keeper/genesis.go

```diff
@@ -22,39 +23,146 @@ func (k Keeper) InitGenesis(ctx sdk.Context, genState *types.GenesisState) {
 		}
 	}
 
+	k.SetValidatorSetUpdateId(ctx, genState.ValsetUpdateId)
+	for _, v2h := range genState.ValsetUpdateIdToHeight {
+		k.SetValsetUpdateBlockHeight(ctx, v2h.ValsetUpdateId, v2h.Height)
+	}
+
+	for _, cccp := range genState.ConsumerAdditionProposals {
+		// prevent implicit memory aliasing
+		cccp := cccp
+		if err := k.SetPendingConsumerAdditionProp(ctx, &cccp); err != nil {
+			panic(fmt.Errorf("pending create consumer chain proposal could not be persisted: %w", err))
+		}
+	}
+	for _, sccp := range genState.ConsumerRemovalProposals {
+		k.SetPendingConsumerRemovalProp(ctx, sccp.ChainId, sccp.StopTime)
+	}
+	for _, ubdOp := range genState.UnbondingOps {
+		if err := k.SetUnbondingOp(ctx, ubdOp); err != nil {
+			panic(fmt.Errorf("unbonding op could not be persisted: %w", err))
+		}
+	}
+
+	if genState.MatureUnbondingOps != nil {
+		if err := k.AppendMaturedUnbondingOps(ctx, genState.MatureUnbondingOps.Ids); err != nil {
+			panic(err)
```

Comment:
At the moment, the list of `MaturedUnbondingOps` is not stored across blocks, i.e., it is emptied in `EndBlock`, see https://github.com/cosmos/interchain-security/blob/6e0666275a219ebb729417557ef76f4c8fe6c10b/x/ccv/provider/module.go#L167. This will change if we implement the sovereign to consumer transition (see [spec](https://github.com/cosmos/ibc/pull/840)). So probably it's good to keep it, but we should add a comment about this.